target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -340,6 +340,8 @@ get_status_led() {
 		status_led="zbt-wg3526:green:status"
 		;;
 	esac
+
+	[ -z "$status_led" ] && status_led="$(get_dt_led status)"
```

Comment:
Please move this change into an extra commit. Maybe it would be better to omit this change for now, add a diag led to the file and migrate the **whole** target to an alias based status led later.